pkg/cluster/internal/kubeadm/config.go

```diff
@@ -525,6 +525,9 @@ mode: "{{ .KubeProxyMode }}"
 {{ range $key := .SortedFeatureGateKeys }}
   "{{ $key }}": {{ index $.FeatureGates $key }}
 {{end}}{{end}}
+iptables:
+  minSyncPeriod: 10s
+  syncPeriod: 60s
```

Comment:
do these match GCE?
should we do it for the other versions?

Reply:
`syncPeriod` is mostly irrelevant these days and should just be left as whatever the default is

`minSyncPeriod: 10s` seems way too high. There is a huge difference between `minSyncPeriod: 0s` and `minSyncPeriod: 1s`, but anything beyond that is mostly just causing iptables updates to lag behind.

Reply:
@aojea did we ever adjust the default upstream?

Reply:
we did https://github.com/kubernetes-sigs/kind/pull/1706

Reply:
That's a link back to this very PR?
I see that https://github.com/kubernetes/kubernetes/pull/92860 landed with a cross reference here and I remember that, but that's not this.

Reply:
Lol I pasted the wrong link 
https://github.com/kubernetes/kubernetes/pull/92836

Reply:
thanks! I thought maybe we had but couldn't find it. glad to see that we did. :-)